karabo_data/geometry2/__init__.py

```diff
@@ -359,10 +368,23 @@ def plot_data_fast(self, data, axis_units='px', frontview=True):
         frontview : bool
           If True (the default), x increases to the left, as if you were looking
           along the beam. False gives a 'looking into the beam' view.
+        ax : `~matplotlib.axes.Axes`, list of Axes, optional
+          Parent axes that will be used to draw the image.
+        figsize : tuple
+          Size of the figure to be drawn (default: (10, 10))
```

Comment:
Should document that it's width/height in "inches" (as defined by matplotlib)

Reply:
Yes